packages/app/src/app/pages/Dashboard/Content/routes/Repositories/index.tsx

```diff
@@ -89,6 +95,13 @@ export const RepositoriesPage = () => {
         showBetaBadge
         nestedPageType={pageType}
       />
+      {!isNotificationDismissed ? (
+        <Notification pageType={pageType}>
+          {itemsToShow().length === 0
+            ? 'CodeSandbox Projects is now Repositories: an improved git workflow powered by the cloud. '
```

Comment:
This text sounds a bit weird to me. What if we take the chance here to explain that Repositories are just Sandboxes with a versioning/git? cc @lenasmayor @necoline

Reply:
Felipe suggested this so we would align with the copy for the newsletter and blog post. I guess the goal is to explain to migrating users (from v2) what is this new section. New users (from v1) will also see the empty state that will cover the features of the new repositories.